dashboard/config/application.rb

```diff
@@ -37,6 +37,12 @@ class Application < Rails::Application
 
     config.middleware.insert_after Rails::Rack::Logger, VarnishEnvironment
     config.middleware.insert_after VarnishEnvironment, FilesApi
+
+    if CDO.throttle_data_apis
```

Comment:
Do we ever want this to be off, outside of Dave's explanation that Redis isn't available in all non-prod environments?

It might be more useful to have adjustable throttling rates in Gatekeeper, but I don't know if that's tricky to set up.